packages/react/src/HumanNameInput/HumanNameInput.tsx

```diff
@@ -22,33 +18,33 @@ export function HumanNameInput(props: HumanNameInputProps): JSX.Element {
   }
 
   function setUse(use: 'temp' | 'old' | 'usual' | 'official' | 'nickname' | 'anonymous' | 'maiden' | undefined): void {
-    setValueWrapper({ ...valueRef.current, use: use || undefined });
+    setValueWrapper({ ...value, use: use || undefined });
   }
 
   function setPrefix(prefix: string): void {
     setValueWrapper({
-      ...valueRef.current,
+      ...value,
```

Comment:
As discussed offline... I think this is fine, and probably an artifact of previous versions.  We just test the heck out of it to be sure.